src/mod/server.mod/server.c

```diff
@@ -1386,6 +1400,10 @@ static tcl_strings my_tcl_strings[] = {
   {"connect-server",      connectserver,  120,               0},
   {"stackable-commands",  stackablecmds,  510,               0},
   {"stackable2-commands", stackable2cmds, 510,               0},
+  {"sasl-mechanism",      sasl_mechanism, 24,                0},
+  {"sasl-username",       sasl_username,  NICKMAX,           0},
+  {"sasl-password",       sasl_password,  80,                0},
+  {"sasl-key",            sasl_key,       120,               0},
```

Comment:
sasl-key isn't in config, should this be -ecdsa? (or add a setting in config?)

Reply:
fixed. renamed eggdrop config setting sasl-external into sasl-key